lib/ecs.go

```diff
@@ -115,8 +115,38 @@ func GetPendingEcsTasksCount(awsSess *session.Session, cluster string) int64 {
 func ListServicesForEcsCluster(awsSess *session.Session, cluster string) []*ecs.Service {
 	svc := ecs.New(awsSess)
 
-	services, err := svc.ListServices(&ecs.ListServicesInput{
+	var allServices []*ecs.Service
+	err := svc.ListServicesPages(&ecs.ListServicesInput{
 		Cluster: aws.String(cluster),
+	}, func(page *ecs.ListServicesOutput, lastPage bool) bool {
+		services, err := DescribeEcsServicesForArns(awsSess, page.ServiceArns, cluster)
+		if err != nil {
+			if aerr, ok := err.(awserr.Error); ok {
```

Comment:
Seems like this `if/else` block is repeated a few times.  Would it be worth factoring it out to its own function?